keel-api/keel-api.gradle.kts

```diff
@@ -10,6 +10,7 @@ dependencies {
   api(project(":keel-eureka"))
   api(project(":keel-artifact"))
 
+  implementation(project(":keel-bakery-plugin"))
```

Comment:
I made this a separate plugin because I think we may want some of it to be in `keel-nflx` but it's also kind of part of ec2